terraform/helm/aptos-node/values.yaml

```diff
@@ -149,8 +149,10 @@ fullnode:
     logger: {}
     mempool:
       shared_mempool_max_concurrent_inbound_syncs: 16 # default 4
-      max_broadcasts_per_peer: 2 # default 1
+      max_broadcasts_per_peer: 4 # default 1
       default_failovers: 0 # default 3
+      shared_mempool_batch_size: 200 # default 100
```

Comment:
Should we make these configs default in Rust instead?

Reply:
The updated config makes sense for vfns where we can be aggressive (since there's only one connection to the validator). Mempool configs in rust would apply to pfns and validators. For example, validators doing large batches to all other validators might overload the network - we'd want to be more careful applying aggressive configs there.